foreman_openscap.gemspec

```diff
@@ -15,5 +15,5 @@ Gem::Specification.new do |s|
   s.test_files = Dir["test/**/*"]
 
   s.add_dependency 'deface', '~> 0'
-  s.add_dependency 'scaptimony', '~> 0'
+  s.add_dependency 'scaptimony', ['>= 0.3.0', '< 0.4.0']
```

Comment:
In fact, the `~> 0.3.0` is needed. The `~> 0.3` has different semantics. That only shows it, how this notation is confusing.